test/runtime/connectivity.go

```diff
@@ -641,8 +641,10 @@ var _ = Describe("RuntimeValidatedConntrackTest", func() {
 			By(fmt.Sprintf("Testing with endpoint configuration option: ConntrackLocal=%s", conntrackLocalOptionMode))
 
 			for _, endpointToConfigure := range endpointsToConfigure {
-				vm.SetAndWaitForEndpointConfiguration(endpointToConfigure, helpers.OptionConntrackLocal, conntrackLocalOptionMode)
-
+				err := vm.SetAndWaitForEndpointConfiguration(
+					endpointToConfigure, helpers.OptionConntrackLocal, conntrackLocalOptionMode)
+				Expect(err).To(BeNil(), "Cannot %q ConntrackLocal to endpoint %q",
```

Comment:
`..."Cannot set ConnTrackLocal=%q for endpoint %s", conntrackLocalOptionMode, endpointToConfigure)`